### ISSUE TITLE
spaced-comment does not correctly ignore JSDoc comments in files with CRLF (Windows) line endings

### ISSUE THREAD

**Issue by DavidAnson** [eslint/eslint#2884]
Reproduce the problem by saving a file that passes this rule with CRLF line-endings.

PR with a fix to follow shortly.

**Comment by nzakas** [comment]
Can you please show an example of the problematic code and show the ESLint output? It's hard for others to understand what you're reporting without such information (and we do specifically ask for this info in our contribution guidelines for bugs). Thanks!

**Comment by DavidAnson** [comment]
I've included an example below. Note that ~identical files are being linted and the one with LF line endings does not trigger the spaced-comment rule while the one with CRLF line endings does. This seems wrong to me (the rule has nothing to do with line endings) and seems attributable to the absence of '\r' in the RegExp I changed as part of the associated pull request.

```
D:\T\eslint-2884>npm install eslint@0.24.0
...

D:\T\eslint-2884>type LF.js
/**
 * This is a function in a file with LF line endings.
 */
function fun() {
}

D:\T\eslint-2884>node_modules\.bin\eslint --rule "strict: 0, no-unused-vars: 0, spaced-comment: 2" LF.js

D:\T\eslint-2884>type CRLF.js
/**
 * This is a function in a file with CRLF line endings.
 */
function fun() {
}

D:\T\eslint-2884>node_modules\.bin\eslint --rule "strict: 0, no-unused-vars: 0, spaced-comment: 2" CRLF.js

CRLF.js
  1:0  error  Expected space or tab after /* in comment  spaced-comment

? 1 problem (1 error, 0 warnings)
```
